.github/workflows/client.yml

```
@@ -37,7 +37,7 @@ jobs:
       env:
         cache-name: cache-yarn-dependencies
       with:
-        # maven dependencies are stored in `~/.m2` on Linux/macOS
+        # npm dependencies are stored in `~/.m2` on Linux/macOS
         path: ~/.npm
```

You probably meant `~/.npm` here?
```suggestion
        # npm dependencies are stored in `~/.npm` on Linux/macOS
        path: ~/.npm
```

---

Oh ... sorry. Yes.